### DISCUSSION TITLE
Saving Changes to Non-Byte-Mapped Views

### DISCUSSION THREAD

**Discussion by toolCHAINZ** [Vector35/binaryninja-api#2777] [API] [answered]
I have a [Binary View](https://github.com/toolCHAINZ/hexfiles) in which the loaded bytes are not in one-to-one correspondence with bytes of the source file. Is there a way to completely override Binja's "save file contents only" functionality for that view?

**Comment by psifertex** [comment, marked as answer]
Sorry for the delay -- looks like there's a perform_save method on the binary view you can likely implement: https://api.binary.ninja/binaryninja.binaryview-module.html#binaryninja.binaryview.BinaryView.perform_save
